src/commands/wallet/create.ts

```diff
@@ -111,28 +111,28 @@ export default class extends Command {
         options.name = options.name.trim();
         const networkHttp = new NetworkHttp(options.url);
         const blockHttp = new BlockHttp(options.url);
-        forkJoin(networkHttp.getNetworkType(), blockHttp.getBlockByHeight(1))
-            .subscribe((res) => {
+        forkJoin(networkHttp.getNetworkType(), blockHttp.getBlockByHeight('1'))
+            .subscribe((res: [NetworkType, BlockInfo]) => {
                 if (res[0] !== networkType) {
                     console.log('The network provided and the node network don\'t match.');
-                } else {
-                    const isWalletExist: boolean = this.walletService.checkWalletExist(options.name);
-                    if (isWalletExist) {
-                        if (!readlineSync.keyInYN('wallet ' + options.name + ' has already exist. Are you sure to recover it?')) {
-                            return;
-                        }
+                    return;
+                }
+                const isWalletExist: boolean = this.walletService.checkWalletExist(options.name);
+                if (isWalletExist) {
+                    if (!readlineSync.keyInYN('wallet ' + options.name + ' has already exist. Are you sure to recover it?')) {
```

Comment:
change text to : `wallet ... already exists. Do you wish to recover this wallet?`

Reply:
fixed